src/index.js

```diff
@@ -4,14 +4,15 @@ import { Store } from './stores/Store';
 import { App } from './components/App';
 import a11y  from 'react-a11y';
 import 'bootstrap/dist/css/bootstrap.css';
-import { SE } from './seed/SE';
 
 const env = process.env.NODE_ENV;
 const store = new Store();
 
-// FOR DEVELOPMENT ONLY,
-// DELETE THIS BEFORE RELEASE
-store.seed(SE);
+fetch('/api/plan/all')
+  .then((response) => response.json())
+  .then((plans) => {
+    store.loadPlan(plans[0]._id);
```

Comment:
instead of seeding with the se plan, now our dirty hack is to get all the plans, then grab the first one. Oh, btw this wont work on your system until you create a plan. sooooo yeah. i did it in mongoose and just deleted it after i had one in there, you do you :+1: